Packages/com.unity.inputsystem/package.json

```diff
@@ -19,5 +19,7 @@
 		"vr",
 		"xr"
 	],
-	"dependencies": {}
+	"dependencies": {
+		"com.unity.ugui": "1.0.0"
```

Comment:
Does this affect the min Unity 2019.1 version we support? Which public version has the change with the empty uGUI package?

Reply:
It took me some digging, but it seems that this was added  in 2019.1.0b2 (making that our minimum supported version)